pkgs/os-specific/linux/minimal-bootstrap/default.nix

```diff
@@ -20,7 +20,9 @@ lib.makeScope
     mes = callPackage ./mes { };
     mes-libc = callPackage ./mes/libc.nix { };
 
-    inherit (callPackage ./stage0-posix { }) kaem m2libc mescc-tools mescc-tools-extra;
+    stage0-posix = callPackage ./stage0-posix { };
+
+    inherit (self.stage0-posix) kaem m2libc mescc-tools mescc-tools-extra;
```

Comment:
`self` not needed as this scope has `with self;`

Reply:
Mmm, I sort of think perhaps then we should use less `with` not less `self.` :D

Reply:
sounds good :)